grai-server/app/connections/tasks.py

```diff
@@ -34,6 +32,8 @@ def execute_run(run: Run):
 
         if connector.name == "PostgreSQL":
             run_postgres(run)
+        if connector.name == "Snowflake":
```

Comment:
It might be a good idea to create an enum with the supported connector names and add them explicitly to the Connector model under [choices](https://docs.djangoproject.com/en/4.1/ref/models/fields/#choices)

Reply:
A good thought